src/DetailsView/components/adhoc-tab-stops-test-view.tsx

```diff
@@ -151,6 +154,14 @@ export const AdhocTabStopsTestView = NamedFC<AdhocTabStopsTestViewProps>(
                         failureInstanceState={props.tabStopsViewStoreData.failureInstanceState}
                         requirementState={requirementState}
                     />
+                    <FocusComponent
+                        windowUtils={props.deps.windowUtils}
```

Comment:
It seems like other components in this file define their own `deps` to easily distinguish between dependencies leveraged in this component and dependencies required by child components. I wonder if we should do the same so that `AdhocTabStopsTestViewDeps` adds an additional `& FocusComponentDeps` constraint?